No rule to make target 'aclk/aclk-schemas/proto/agent/v1/connection.proto'

Hej,

since a couple of days netdata failed to run the daily updates via CRON.
I manually reinstalled netadata to reproduce the error.

##### Bug report summary
```
./kickstart.sh --reinstall --disable-telemetry --non-interactive > kickstart.log 2>&1
[...]
 --- Compile netdata ---
[/tmp/netdata-kickstart-aGo567vfp2/netdata-v1.31.0-118-gfd9e87e8a]# make -j12 
make: *** No rule to make target 'aclk/aclk-schemas/proto/agent/v1/connection.proto', needed by 'aclk/aclk-schemas/proto/agent/v1/connection.pb.cc'.  Stop.
 FAILED
```

Complete log attached:
[kickstart.log](https://github.com/netdata/netdata/files/6814240/kickstart.log)

##### OS / Environment
```
Linux enterprise 5.12.14-x86_64 #1 SMP Mon Jul 5 19:41:14 CEST 2021 x86_64 Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz GenuineIntel GNU/Linux
/etc/gentoo-release:Gentoo Base System release 2.7
/etc/os-release:NAME=Gentoo
/etc/os-release:ID=gentoo
/etc/os-release:PRETTY_NAME="Gentoo/Linux"
/etc/os-release:ANSI_COLOR="1;32"
/etc/os-release:HOME_URL="https://www.gentoo.org/"
/etc/os-release:SUPPORT_URL="https://www.gentoo.org/support/"
/etc/os-release:BUG_REPORT_URL="https://bugs.gentoo.org/"
```

##### Netdata version
The currently running and last version that updated itself was:
```
Version: netdata v1.31.0-101-nightly
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--with-bundled-lws' 'CFLAGS=-O2' 'LDFLAGS='
Features:
    dbengine:                YES
    Native HTTPS:            YES
    Netdata Cloud:           YES 
    ACLK Next Generation:    YES
    ACLK Legacy:             YES
    TLS Host Verification:   YES
Libraries:
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  YES
    libcrypto:               YES
    libm:                    YES
    LWS:                     YES static v3.2.2
    mosquitto:               YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    YES
    IPMI:                    NO
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
```

The version which fails to compile is:
`v1.31.0-118-gfd9e87e8a`

(The Gentoo system has been updated in-between, so maybe there is some version incompatibility for a lib which netdata depends on...)

##### Installation method
`./kickstart.sh --reinstall --disable-telemetry --non-interactive > kickstart.log 2>&1`

##### Component Name
netdata

##### Steps To Reproduce
1. Just try to install or reinstall netdata via any method

##### Expected behavior
Netdata should compile.


Thanks for your help,
Bjoern

--- a/Makefile.am
+++ b/Makefile.am
@@ -57,6 +57,7 @@ dist_noinst_DATA = \
     contrib \
     docs \
     mqtt_websockets \
+    aclk/aclk-schemas \
     netdata.cppcheck \
     netdata.spec \
     package.json \

